README.md

```diff
@@ -30,6 +30,18 @@ It will listen on <http://127.0.0.1:3030> by default.
 Note: If you have issues with `npm install` please update to npm 2.0+ using `npm install -g npm@2` 
 ([Issue #1594](https://github.com/mozilla/fxa-content-server/issues/1594))
 
+## Docker Based Development
+
+To run the content server via Docker, three steps are required:
+
+    $ docker build --rm -t mozilla/fxa_content_server
+    $ docker run --rm -v $PWD:/opt/fxa mozilla/fxa_content_server npm install
+    $ docker run --rm -v $PWD:/opt/fxa --net=host mozilla/fxa_content_server
+
+This method shares the codebase into the running container so that you can install npm and various modules required by package.json. It then runs FxA content server in a container, while allowing you to use your IDE of choice from your normal desktop environment to develop code.
+
+Be sure to copy server/config/local.json-dist to server/config/local.json per usual before the final docker invocation to run the service.
```

Comment:
Thanks for pointing that out, I'll incorporate that into the readme (for this and the other repos) and update the various pull requests. The fully synchronized version that I mentioned will handle these things in a more consistent way.